src/tracking/particles.cpp

```diff
@@ -68,13 +69,16 @@ namespace impactx
 
         }
 
-        amrex::ParmParse const pp_algo("algo");
-        bool space_charge = false;
-        pp_algo.query("space_charge", space_charge);
+        auto space_charge = get_space_charge_algo();
         if (verbose > 0) {
-            amrex::Print() << " Space Charge effects: " << space_charge << "\n";
+            amrex::Print() << " Space Charge effects: " << amrex::getEnumNameString(space_charge) << "\n";
```

Comment:
I think I want to strip the `True_` prefix with a helper function, because then we exactly mirror the input we receive from the user when we print to the terminal.

Reply:
Done.